ruleset/decoders/0007-wazuh-api_decoders.xml

```diff
@@ -5,16 +5,27 @@
 <!-- Sample log
 2021/04/20 16:00:35 INFO: wazuh 127.0.0.1 "PUT /agents/group" with parameters {"group_id": "group1", "agents_list": "629,650,654,682"} and body {} done in 0.075s: 200
 2021/10/05 10:30:21 INFO: Generated private key file in WAZUH_PATH/api/configuration/ssl/server.key
+2022/02/03 10:37:36 INFO: wazuh (d8466023fdec3f1310679989d8827eee) 172.20.0.1 "GET /rules" with parameters {"filename": "0010-rules_config.xml"} and body {} done in 0.073s: 200
 -->
 
 <decoder name="wazuh-api">
    <prematch type="pcre2">\s\d+:\d+:\d+\s\w+:\s\S+\s\d+.\d+.\d+.\d+</prematch>
 </decoder>
 
+<decoder name="wazuh-api">
+   <prematch type="pcre2">\s\d+:\d+:\d+\s\w+:\s\S+\s\(\S+\)\s+\d+.\d+.\d+.\d+</prematch>
+</decoder>
+
 <decoder name="wazuh-api-info">
   <prematch type="pcre2">^\d+\/\d+\/\d+\s\d+:\d+:\d+\sINFO:\s\w+\s\w+|^\d+\/\d+\/\d+\s\d+:\d+:\d+\sWARNING:\s\w+\s\w+|^\d+\/\d+\/\d+\s\d+:\d+:\d+\sERROR:\s\w+\s\w+|^\d+\/\d+\/\d+\s\d+:\d+:\d+\sCRITICAL:\s\w+\s\w+</prematch>
 </decoder>
 
+<decoder name="wazuh-api-fields">
+  <parent>wazuh-api</parent>
+  <regex type="pcre2">\S+\s\((\S+)\)\s\d+\.\d+\.\d+\.\d+\s</regex>
+  <order>hash</order>
```

Comment:
`hash` is not really descriptive. I would call this field `auth_context_hash`

```suggestion
  <order>auth_context_hash</order>
```

Ref.: https://documentation.wazuh.com/current/user-manual/api/rbac/auth-context.html